Source/Controllers/SPAppController.m

```diff
@@ -705,6 +705,34 @@ - (void)openUserBundleAtPath:(NSString *)filePath
 		}
 	}
 
+	SPLog(@"cmdData %@", cmdData);
+
+	// first lets check if it's a legacy bundle
+	if([newPath containsString:SPUserBundleFileExtension] == YES){
```

Comment:
Perhaps instead of SPUserBundleFileExtension and SPUserBundleFileExtensionV2 we could convert the current SPUserBundleFileExtension into SPUserBundleLegacyFileExtension and allow SPUserBundleFileExtensionV2 to take the name SPUserBundleFileExtension?